extension/src/test/LanguageFunctions.test.ts

```diff
@@ -1198,6 +1199,15 @@ suite("ALObject TransUnit Tests", function () {
       assert.fail("No transunits identified");
     }
   });
+  test("findSourceOfTranslatedTexts with custom note", async function () {
+    let uris: Uri[] = await workspace.findFiles("**/customNotes.xlf");
```

Comment:
Put the file in a new subfolder to src\test\resources instead and use `path.resolve(__dirname, 'path to your xlf file')`, so we do not mess with other tests.

Reply:
Search for `path.resolve(__dirname` and you'll find a few samples